charts/htsget-rs/templates/certificate.yaml

```
@@ -0,0 +1,43 @@
+{{- if .Values.tls.enabled }}
+{{- if or .Values.tls.clusterIssuer .Values.tls.issuer }}
+{{- if and .Values.tls.clusterIssuer .Values.tls.issuer }}
+          {{- fail "Only one of tls.issuer or tls.clusterIssuer should be set" }}
+{{- end -}}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ template "htsget-rs.fullname" . }}-certs
+spec:
+  # Secret names are always required.
+  secretName: {{ template "htsget-rs.fullname" . }}-certs
+
+  duration: 2160h # 90d
+
+  # The use of the common name field has been deprecated since 2000 and is
+  # discouraged from being used.
+  commonName: {{ template "htsget-rs.fullname" . }}
+  isCA: false
+  privateKey:
+    algorithm: ECDSA
+    size: 256
```

```suggestion
    algorithm: Ed25519
```

Use Ed25519 cerfificates or at least 384 byte keys if you use ECDSA